extensions/wikia/PortableInfobox/PortableInfobox.i18n.php

```diff
@@ -17,6 +17,8 @@
 	'portable-infobox-xml-parse-error-tag-name-mismatch' => 'Opening and ending tag mismatch',
 	'portable-infobox-xml-parse-error-tag-not-finished' => 'Premature end of tag',
 	'portable-infobox-xml-parse-error-infobox-tag-attribute-unsupported' => 'Attribute "$1" is not supported in <infobox> tag',
+	// TODO: set proper copy
+	'portable-infobox-unsupported-color-format' => 'Unsupported color format. Currently, only hexadecimal values are supported.'
```

Comment:
I would remove the word "currently" and add an example. Not everyone knows what _hexadecimal_ means.